src/pseudo/NormConservingPsp.jl

```diff
@@ -97,6 +105,44 @@ function eval_psp_energy_correction end
 eval_psp_energy_correction(psp::NormConservingPsp, n_electrons) =
     eval_psp_energy_correction(Float64, psp, n_electrons)
 
+"""
+    eval_psp_density_valence_real(psp, r)
+
+Evaluate the atomic valence charge density in real space.
+"""
+eval_psp_density_valence_real(psp::NormConservingPsp, r::AbstractVector) = 
+    eval_psp_density_valence_real(psp, norm(r))
+
+"""
+    eval_psp_density_valence_fourier(psp, q)
+
+Evaluate the atomic valence charge density in reciprocal space:
+ρval(q) = ∫_{R^3} ρval(r) e^{-iqr} dr
+        = 4π ∫_{R+} ρval(r) sin(qr)/qr r^2 dr
+"""
```

Comment:
these formulas appear several times, maybe merge them in a big comment somewhere?